plugin/pki/path_venafi_cert_enroll.go

```diff
@@ -133,7 +134,7 @@ func (b *backend) pathVenafiIssue(ctx context.Context, req *logical.Request, dat
 		return logical.ErrorResponse("role key type \"any\" not allowed for issuing certificates, only signing"), nil
 	}
 
-	return b.pathVenafiCertObtain(ctx, req, data, role, false)
+	return b.pathVenafiCertObtain(ctx, req, data, role, roleName, false)
```

Comment:
should the rolename be into the role struct in order to avoid the modification of the signature function?

Reply:
It would make sense. I didn't consider it in the scope of this PR since it would require further information to see it doesn't impact other places and would impact to add it. But thank you for pointing out this. We may want to refactor it in another PR.

Reply:
Adding the roleName to the role structure should not impact any functionality at all. It is an addition. On the other hand, modifying the signature of a function already in place bears more risk to have unknown side-effects. If the role name is necessary, lets add it to the proper structure.